spec/requests/api/v1/bulk_export_controller_spec.rb

```diff
@@ -0,0 +1,214 @@
+require 'spec_helper'
+require 'fileutils'
+require 'zip'
+
+describe Api::V1::BulkExportController do
```

Comment:
This should test each api route's if branches making sure everything works correctly.

Although, we are technically using factory bulk_export records here and not really checking the correct data processed (focusing only on response statuses)

I think it should be sufficient for the controller tests though. Given that this seems to be doing a lot of repetitive logic with the other bulk_controller, I am planning on writing a lib so that we just call that in the controller instead of doing the business logic in this controller (and then add the thorough data processed testing there)